src/main/java/org/storage/metadata/controller/UserController.java

```diff
@@ -54,43 +46,4 @@
             return new ResponseEntity<>(null, HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
did you mean to return List<User> here instead of null

Reply:
No, I am returning the list in the try block